app/src/main/res/layout/fragment_adyen_error_top_up.xml

```diff
@@ -0,0 +1,123 @@
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/activity_adyen_error_view"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    >
+
+  <com.airbnb.lottie.LottieAnimationView
+      android:id="@+id/validation_success_animation"
+      android:layout_width="66dp"
+      android:layout_height="66dp"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toTopOf="@+id/guideline4"
+      app:lottie_autoPlay="true"
+      app:lottie_enableMergePathsForKitKatAndAbove="true"
+      app:lottie_rawRes="@raw/adyen_error_animation"
+      />
+
+  <TextView
+      android:id="@+id/error_title"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:layout_marginTop="23dp"
+      android:fontFamily="sans-serif"
+      android:letterSpacing="0.03"
+      android:lineSpacingExtra="4sp"
+      android:text="@string/error_general"
+      android:textAlignment="center"
+      android:textColor="@color/btn_light_enabled_color"
+      android:textSize="20sp"
+      android:textStyle="bold"
+      android:visibility="gone"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toBottomOf="@+id/validation_success_animation"
+      tools:visibility="visible"
+      />
+
+  <TextView
+      android:id="@+id/error_message"
+      style="@style/TextAppearance.AppCompat.Body1.Roboto.Regular"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:layout_marginTop="@dimen/big_margin"
+      android:textAlignment="center"
+      android:textColor="@color/pending_payment_text_color"
+      android:textSize="16sp"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toBottomOf="@+id/error_title"
+      tools:text="@string/purchase_card_error_invalid_details"
+      />
+
+  <TextView
+      android:id="@+id/contact_us"
+      style="@style/TextAppearance.AppCompat.Body1.Roboto.Regular"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:layout_marginTop="42dp"
+      android:text="@string/error_contac_us_body"
+      android:textAlignment="center"
+      android:textColor="@color/grey_8a_alpha"
+      android:textSize="12sp"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toBottomOf="@+id/error_message"
+      />
+
+  <LinearLayout
+      android:id="@+id/layout_support"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:layout_marginTop="18dp"
+      android:gravity="center"
+      android:orientation="horizontal"
+      app:layout_constraintEnd_toEndOf="parent"
```

Comment:
Same comment as the landscape layout regarding the additional linear layout not being needed